samples/ImplicitFlow/AuthorizationServer/.vscode/launch.json

```diff
@@ -0,0 +1,41 @@
+{
+    "version": "0.2.0",
```

Comment:
I know this sample is generated by VS Code, but the spacing doesn't match the 2-space convention used by the other JSON files.